build/sub_logger.go

```diff
@@ -30,10 +29,12 @@ type SubLoggerManager struct {
 var _ LeveledSubLogger = (*SubLoggerManager)(nil)
 
 // NewSubLoggerManager constructs a new SubLoggerManager.
-func NewSubLoggerManager(w io.Writer) *SubLoggerManager {
+func NewSubLoggerManager(handlers ...Handler) *SubLoggerManager {
 	return &SubLoggerManager{
-		loggers:   SubLoggers{},
-		genLogger: btclog.NewBackend(w),
+		loggers: SubLoggers{},
+		genLogger: newSubLogGenerator(
+			newHandlerSet(btclog.LevelInfo, handlers...),
```

Comment:
We have the global variable `LogLevel` that's being set with build tags for testing/development. I assume that still works? Or do we need to use that level here to set the correct default/initial level?

Reply:
that will still work. The previous "NewBackend" also used to set Info level as default. So SetLevel is used later to set these :)